submission/persistence-filesystem/src/main/kotlin/ac/uk/ebi/biostd/persistence/filesystem/api/FtpService.kt

```diff
@@ -0,0 +1,9 @@
+package ac.uk.ebi.biostd.persistence.filesystem.api
+
+import ebi.ac.uk.extended.model.ExtSubmission
+
+interface FtpService {
+    fun processSubmissionFiles(submission: ExtSubmission)
+
+    fun createFtpFolder(relPath: String)
```

Comment:
I think this operation should receive `submission: ExtSubmission` to keep abstractions at the same level.

Reply:
There's an endpoint that uses the submission's relPath in order to force the ftp link creation. This was kept like that to keep compatibility with that endpoint.

https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/web/resources/FtpResource.kt#L17

Otherwise, we would need to change the signature.